org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/thingTranslator.js

```diff
@@ -161,6 +165,24 @@ class Translator {
         });
     }
 
+    _subscribe(deviceId) {
```

Comment:
In the current design, the callback URL are set in the SmartApp as last time when I had the conversation with Avneet, he mentioned that we will not be dynamically changing the callback URL at the current stage. However, the current SmartApp design does provide a option to update the callback URL through a REST API call if needed.